src/main/java/org/springframework/data/tarantool/core/TarantoolTemplate.java

```diff
@@ -429,6 +497,21 @@ public TarantoolConverter getConverter() {
         return converter;
     }
 
+    @Override
+    public void truncate(String spaceName) {
+        //FIXME implement truncate in cartridge-driver and remove this temporary hack
```

Comment:
Add an issue to cartridge-driver

Reply:
https://github.com/tarantool/cartridge-java/issues/93